pylmod/membership.py

```diff
@@ -0,0 +1,94 @@
+"""
+
```

Comment:
Again, please actually describe the module here, something as simple as "This is the module that contains the Membship class for use with the LMOD Membership API (url to docs)"